graphqlapi/src/main/resources/schemafirst/cql_directives.graphql

```diff
@@ -121,7 +121,23 @@ directive @cql_update on FIELD_DEFINITION
 Indicates that a mutation should be mapped to a CQL DELETE query.
 Note that this is not required if the mutation name starts with `delete` or `remove`.
 """
-directive @cql_delete on FIELD_DEFINITION
+directive @cql_delete(
+    """
+    The name of the type to delete. This is only needed if the mutation takes individual key fields
+    as arguments (as opposed to an instance of the type).
+    This must be a type that maps to a table.
+    """
+    targetEntity: String
+
+    """
+    What to do if the entity does not exist. If `false`, the operation will fail. If `true`, it
```

Comment:
what does it mean that `the operation will fail.`? What will be returned to the caller in this case?

Reply:
Actually that description is wrong, the operation will always succeed even if the row does not exist. I'll fix it.